app/views/catalog/_search_sidebar.html.erb

```diff
@@ -1,3 +1,4 @@
+<% # this partial is deprecated with BL8.  catalog/index now renders the sidebar component %>
```

Comment:
We should just delete this partial entirely as part of this PR rather than keep it around unused in the codebase.

Reply:
Done